builders/tutorials/subsquid.md

```diff
@@ -0,0 +1,490 @@
+
+```ts
+    .setDataSource({
+        // Lookup archive by the network name in Subsquid registry
+        // See https://docs.subsquid.io/evm-indexing/supported-networks/
+        chain: {
+            url: assertNotNull(process.env.RPC_ENDPOINT),
+            rateLimit: 300
+        }
+    })
+```
+
+You'll need to provide the RPC URL for your ContainerChain in your `.env` file. Make sure that any existing RPC URLs are removed. If you're using the Demo EVM ContainerChain, the respective line in your .env file will look like this: 
+
+```
+RPC_ENDPOINT={{ networks.dancebox.rpc_url }}
+```
+
+Of course, we need to tell the Subsquid processor which contract we're interested in. Create a constant for the address in the following manner: 
+
+```
+export const CONTRACT_ADDRESS = 'INSERT_CONTRACT_ADDRESS'.toLowerCase();
+```
+
+The `.toLowerCase()` is critical because the Subsquid processor is case sensitive, and some block explorers format contract addresses with capitalization. Now, let's define the event that we want to index by adding the following: 
+
+```ts
+    .addLog({
+        address: [CONTRACT_ADDRESS],
+        topic0: [erc20.events.Transfer.topic],
+        transaction: true,
+    })
+```
+
+The `Transfer` event is defined in `erc20.ts` which was auto-generated when `sqd typegen` was run. The import `import * as erc20 from './abi/erc20'` is already included as part of the Squid EVM template. 
+
+Block range is an important value to modify to narrow the scope of the blocks you're indexing. For example, if you launched your ERC-20 at block `650000`, there is no need to query the chain before that block for transfer events. Setting an accurate block range will improve the performance of your indexer. You can set the earliest block to begin indexing in the following manner: 
+
+```
+    .setBlockRange({from: 632400,})
+```
+
+The chosen start block here corresponds the relevant block to begin indexing on the Demo EVM ContainerChain, but you should change it to one relevant to your ContainerChain and indexer project. 
+
+Change `setFields` section to specify the following data for our processor to ingest:
+
+```ts
+    .setFields({
+        log: {
+            topics: true,
+            data: true,
+        },
+        transaction: {
+            hash: true,
+        },
+    })
+```
+
+We also need to add the following imports to our `processor.ts` file: 
+
+```ts
+import {Store} from '@subsquid/typeorm-store'
+import * as erc20 from './abi/erc20'
+```
+
+Once you've completed the prior steps, your `processor.ts` file should look similar to this:
+
+??? code "View the complete script"
```

Comment:
if you're going to say "should look similar to this" then I feel like the snippet should be expanded, so this can be `???+` instead

Reply:
although you have built up/pieced together the processor.ts file changes, it almost makes more sense to name this "processor.ts" instead of "View the complete script"  just for consistency on this page